Multisig/UI/App/UIKit/CollectiblesViewController/CollectibleTableViewCell.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  CollectibleTableViewCell.swift
+//  Multisig
+//
+//  Created by Andrey Scherbovich on 02.11.20.
+//  Copyright © 2020 Gnosis Ltd. All rights reserved.
+//
+
+import UIKit
+
+class CollectibleTableViewCell: UITableViewCell {
+    @IBOutlet weak var cellWrapperView: UIView!
+    @IBOutlet weak var cellImageView: UIImageView!
+    @IBOutlet weak var cellNameLabel: UILabel!
+    @IBOutlet weak var cellDescriptionLabel: UILabel!
+
+    let cornerRadius: CGFloat = 8
+
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        backgroundColor = .gnoSystemWhite
+
+        cellWrapperView.backgroundColor = .gnoSystemWhite
```

Comment:
Potential bad UI graphical performance because of use of layers. Instead of this, I think, it is better to use Image view with the 'shadow' background - we have such image in the assets folder.